app/pages/patientdata/patientdata.less

```diff
@@ -33,16 +33,21 @@
 
 .patient-data-subnav-text {
   display: inline-block;
-  padding: @patient-data-subnav-vertical-padding @patient-data-subnav-horizontal-padding;
+  padding: @patient-data-subnav-vertical-padding 0 @patient-data-subnav-horizontal-padding 0;
```

Comment:
There's a funny thing going one when the app is in the loading data state (title bar gets really vertically skinny), and I think this change might be the root?
![screenshot 2015-01-13 10 19 44](https://cloud.githubusercontent.com/assets/1588547/5726139/f5f1965c-9b0d-11e4-859f-daa750ee84a0.png)

To get a good long look at it you can use "mock" mode and and the URL http://localhost:3000/?api.patientdata.get.delay=5000#/patients/11/data (auth.skip in mock mode seems to be broken though...)